api/python/provisioner/api.py

```diff
@@ -414,6 +414,34 @@ def get_swupgrade_info(release: str = None, iso_path: str = None,
                      nowait=nowait)
 
 
+def check_iso_authenticity(iso_path: str, sig_file: str,
+                           gpg_pub_key: str = None,
+                           import_pub_key: bool = False, nowait=False):
+    """
+    Validate SW Upgrade single ISO authenticity using GPG tool.
+
+    Parameters
+    ----------
+    iso_path: str
+        Path to SW upgrade single ISO bundle
+    sig_file: str
+        Path to file with ISO signature
+    gpg_pub_key: str
+        (Optional) Path to the custom GPG public key
+    import_pub_key: bool
```

Comment:
Could you please be specific what is the difference between:
gpg_pub_key
import_pub_key

Should it help if we assume that `import_pub_key`, if specified, also means `import_pub_key` to be true.

Reply:
Option `gpg_pub_key` is needed more for testing purposes.  It means that by default we assume that Seagate's GPG public key is imported. I provided the `import_pub_key` option to import provided `gpt_pub_key` if needed. Yes, it is correct: if `--import-pub-key` is specified, it means that `import_pub_key` is set to `True`